script/tasks/diamondUpdateFacet.sh

```
@@ -121,10 +121,17 @@ diamondUpdateFacet() {
           CLEAN_RETURN_DATA=$(echo $RAW_RETURN_DATA | sed 's/^.*{\"logs/{\"logs/')
           FACET_CUT=$(echo $CLEAN_RETURN_DATA | jq -r '.returns.cutData.value')
 
-          if [ "$FACET_CUT" != "0x" ]; then
-            echoDebug "Proposing facet cut for $SCRIPT..."
-            DIAMOND_ADDRESS=$(getContractAddressFromDeploymentLogs "$NETWORK" "$ENVIRONMENT" "$DIAMOND_CONTRACT_NAME")
-            npx tsx script/deploy/safe/propose-to-safe.ts --to "$DIAMOND_ADDRESS" --calldata "$FACET_CUT" --network "$NETWORK" --rpcUrl "$(getRPCUrl $NETWORK)" --privateKey "$SAFE_SIGNER_PRIVATE_KEY"
+          if [ "$FACET_CUT" == "0x" ] || [ -z "$FACET_CUT" ]; then
+            error "Unable to extract facet cut data from RPC response at logs.returns.cutData.value"
+          else
+            if [ "$DEPLOY_NEW_NETWORK_MODE" == "true" ]; then
+              echo "DEPLOY_NEW_NETWORK_MODE is activated - executing facet cut for $script on network $NETWORK..."
```

Shellcheck (0.10.0)

[warning] script is referenced but not assigned (did you mean 'SCRIPT'?).

(SC2154)